CMakeLists.txt

```diff
@@ -192,12 +192,12 @@ include_directories(third_party/inih/cpp)
 
 # abseil-cpp
 set(CMAKE_CXX_STANDARD 17)
-add_subdirectory(third_party/abseil-cpp)
+add_subdirectory(third_party/abseil-cpp EXCLUDE_FROM_ALL)
```

Comment:
why add this `EXCLUDE_FROM_ALL` flag?

Reply:
https://cmake.org/cmake/help/v3.14/prop_tgt/EXCLUDE_FROM_ALL.html
only build what we want of absl by dependency